nss-day-cohort-39/nutshell-zaffre-zorillas
# list of user's events now rendering on home page

### Changes
- Performed fetch call to LH:3000 for events in database.json
- Composed html for individual events listing name, date, and location
- Filtered through all events to find events created by single user, and rendered to DOM

### Testing 
- Run json-server -p 3000 -w database.json from src>api
- Serve from src to LH:5000
- Make sure Welcome Page renders at the top, with register and login buttons
- Make sure two events (Cam Concert & Rachel's Birthday) render to page
